pkg/awsutils/awsutils.go

```diff
@@ -674,6 +675,16 @@ func (cache *EC2InstanceMetadataCache) tagENI(eniID string) {
 		})
 	}
 
+	//additionalEniTags for adding additional tags on ENI
+	additionalEniTags := os.Getenv(additionalEniTagsEnvVar)
+	if additionalEniTags != "" {
+		tagsMap, err := parseAdditionalEniTagsMap(additionalEniTags)
+		if err != nil {
+			log.Warnf("Failed to add additional tags the newly created ENI %s: %v", eniID, err)
+		}
+		tags = mapToTags(tagsMap)
```

Comment:
AFAICT, this will remove the tag for the node name (instance ID) added above on lines 661-666

Reply:
Thanks for catching.